CDP4Common/Helpers/PropertyDescriptor.cs

```diff
@@ -0,0 +1,246 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="PropertyDescriptor.cs" company="RHEA System S.A.">
+//    Copyright (c) 2015-2023 RHEA System S.A.
+//
+//    Author: Sam Gerené, Alex Vorobiev, Alexander van Delft, Nathanael Smiechowski, 
+//            Antoine Théate, Omar Elabiary, Jaime Bernar
+//
+//    This file is part of CDP4-COMET SDK Community Edition
+//    This is an auto-generated class. Any manual changes to this file will be overwritten!
+//
+//    The CDP4-COMET SDK Community Edition is free software; you can redistribute it and/or
+//    modify it under the terms of the GNU Lesser General Public
+//    License as published by the Free Software Foundation; either
+//    version 3 of the License, or (at your option) any later version.
+//
+//    The CDP4-COMET SDK Community Edition is distributed in the hope that it will be useful,
+//    but WITHOUT ANY WARRANTY; without even the implied warranty of
+//    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+//    Lesser General Public License for more details.
+//
+//    You should have received a copy of the GNU Lesser General Public License
+//    along with this program; if not, write to the Free Software Foundation,
+//    Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace CDP4Common.PropertyAccesor
+{
+    using System;
+    using System.Text.RegularExpressions;
+
+    /// <summary>
+    /// The <see cref="PropertyDescriptor"/> is used ...
```

Comment:
I would have a better description of this PropertyDescriptor class

Reply:
thanks for spotting that, its updated